### ISSUE TITLE
 Can't load plugin: sqlalchemy.dialects:drill.jdbc

### ISSUE THREAD

**Issue by zachary124** [apache/incubator-superset#8151]
hello，I use drill on superset。when I use drill+sadrill(REST API) I can work in SQL LAB，but not add table and it tell me table not found,I think the REST not get meta schema so not work。
then I try  to use drill+jdbc and i had installed python package pydrill and I get a error like  Can't load plugin: sqlalchemy.dialects:drill.jdbc，what can I do for this？thanks very much。

The document said：
Once that is done, you can connect to Drill in two ways, either via the REST interface or by JDBC. If you are connecting via JDBC, you must have the Drill JDBC Driver installed.

 I am confused iwhat driver I should install。

the version I pull from git master。

**Comment by villebro** [comment]
@cgivre can you help out?

**Comment by cgivre** [comment]
@zachary124 
To use Drill with JDBC with Superset you have to use our sqlalchemy dialect which is available here:https://github.com/cgivre/sqlalchemy-drill/tree/jdbc_support

So, I've pushed a branch that has JDBC support, however this is a WIP.  It works, however, @JohnOmernik and I are trying to figure out how to best handle downloading, installing and configuring the JDBC driver as a part of the install process.  

The code on the repo works but at the moment, it has hard-coded paths to the JDBC driver.  You can see how to configure it here: https://github.com/cgivre/sqlalchemy-drill/blob/jdbc_support/test/test_jdbc.py

I'll try to fix it up tonight and remove the hard coded links.

**Comment by zachary124** [comment]
@cgivre  hello， thank you for your reply, I had clone your code and run the test_jdbc.py,  it drill+sadrill can work,but the jdbc get error about ：
TypeError: function takes exactly 4 arguments (3 given)
then I modify the url is
 drill+jdbc://admin:password@localhost:31010/mongo.test_db 
I still get a error about:

TypeError: connect() got an unexpected keyword argument 'database'

and I try in superset is same error。

**Comment by cgivre** [comment]
@zachary124 Can you try it without the database?  IE:
drill+jdbc://admin:password@localhost:31010/

**Comment by zachary124** [comment]
@cgivre  yes,I  used it first drill+jdbc://admin:password@localhost:31010 but get error:
TypeError: function takes exactly 4 arguments (3 given)

**Comment by zachary124** [comment]
@cgivre The dependencies were reinstalled and the test_jdbc.py can work 。but in superset has another issue,I will create new issue in issue list, thank you.
